### ISSUE TITLE
aerothermal flux value used to jettison fairing in ascent guidance probably incorrect

### ISSUE THREAD

**Issue by Alioth81** [MuMech/MechJeb2#1505]
The internal value of the aerothermal flux entry field in the ascent guidance to jettison the fairing is probably not correct.
If set to the default value 1135 W/m^2 the fairing is jettisoned just at the edge of space practically in vaccuum which is most likely wrong.

If set to a value 1000x higher the fairing would be jettisoned at much lower altitude somewhere below 30km and aerothermal flux is not longer the limiting factor (in this case it will use the default min height).

This leads me to believe that there an error in calculating this value potentially as simple a a unit conversion.

**Comment by Capkirk123** [comment]
The aerothermal flux value is tuned for RSS/RO, and using the aerothermal flux limits from the Thor-Agena payload guide. If you wish for the fairing to eject earlier in stock, you can decrease the aerothermal flux limit in the settings.

**Comment by Alioth81** [comment]
Thanks for the clarification so it is intended to be this way.
You suggest a decrease but I found an increase of the limit is actually producing the effect of lowering the jettison height. Maybe I misunderstood you.

Isi it possible to have a different default for stock if possible? Or have it disabled by default? 

I am not affected anymore because I changed the value but for others it might be confusing as the actual value of thermal flux is not displayed in the UI (even in debug modus as far as I can see). Therefore it was not obvious to me that I have to increase the value by a factor of about 1000x of the default to have a real effect (this will set it somewhere below 30km) the default is practically in space.
